src/js/popup/popup.js

```diff
@@ -269,15 +237,16 @@ async function choosePanel(numRemaining, panelId, premium, premiumSubdomainSet){
     if (premiumWrapper.length) {
       premiumWrapper[0].remove();
     }
-    return numRemaining === 0 ? "maxAliasesPanel" : `panel${panelId}`;
+
+    return `panel${panelId}`;
   }
 }
 
 function checkUserSubdomain(premiumSubdomainSet){
   const educationalComponent = document.querySelector(".educational-component");
   const registerDomainComponent = document.querySelector(".register-domain-component");
 
-  if (premiumSubdomainSet !== "None") {
+  if (premiumSubdomainSet === "None") {
```

Comment:
The logic here needs to keep the `!`. 

I tested with two accounts, both premium email (not phones). One had a subdomain registered and one one did not. When the domain was registered, it kept prompting me to register one. 

I'm going to flip this back in a commit and then mark it as good to go for now. Any additional fixes can be caught in post!